less/elements.less

```diff
@@ -161,9 +165,12 @@ item-list-controls search {
     display: flex;
     align-items: center;
     padding: .5rem 0;
+    align-self: stretch;
+    align-items: stretch;
   }
 
   .filter-control {
+    height: 100%;
```

Comment:
These styles are just increasing the click area of these controls to make them larger click targets, without affecting their appearances.

Reply:
Also increases the height of the border so that it touches the edges now.

<img src="https://github.com/foundryvtt/dnd5e/assets/207433/447c6308-4c98-46ed-b41d-0c413cfdf65f"> <img src="https://github.com/foundryvtt/dnd5e/assets/207433/4fcc727b-f9e2-4da4-92a5-96cd5d2694a3">

It's fine though, I think it probably looks better anyway.